sdk/src/main/java/com/ingresse/sdk/v2/parses/ResponseParser.kt

```diff
@@ -60,6 +62,11 @@ private fun <T> parseEmptyBodyError(): Result<T> {
 private fun <T> Gson.parseIngresseError(body: String): Result<T> {
     val result = fromJson(body, IngresseError::class.java)
     val error = result.responseError
+
+    if (Errors.tokenError.contains(error.code)) {
```

Comment:
@ingresse/backend Aqui é onde eu trato tudo que é geral para as chamadas da api (200). Conseguem olhar com calma para ver se consigo adicionar alguma regra importante aqui? Se for regra de outro tipo de resposto, daí criarei outro _Parser_ para não ficar um amontoado de regras nesse.

Reply:
Eu acredito que, em algum momento, faça sentido refatorar essa classe em 2, uma pra respostas aceitas e outra para erros. Acredito que por enquanto ela está ok.

Reply:
Vou ter que fazer uma revisão nesse PR já que o iOS me deu uns _insights_, daí podemos até discutir um pouco mais do roteamento padrão de erros, que tal?!